doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1026,6 +1026,59 @@ Alerts and notifications
 Default notification should have `text-info` color. Success notification should
 have `text-success` color. Failure notification should have `text-danger` color.
 
+Accessibility
+.............
+
+Many parts of the Ceph Dashboard are modeled on `Web Content Accessibility Guidelines (WCAG) 2.1 <https://www.w3.org/TR/WCAG21/>`_  level A accessibility conformance guidelines. 
+By implementing accessibility best practices, you are improving the usability of the Ceph Dashboard for blind and visually impaired users.
+
+**Accessibility checkers:** During development, you can test the accessibility compliance of your features using one of the tools below:
+
+- `Accessibility insights plugin <https://accessibilityinsights.io/downloads/>`_
+- `Site Improve plugin <https://www.siteimprove.com/integrations/browser-extensions/>`_
+- `Axe devtools <https://www.deque.com/axe/devtools/>`_
+
+Testing with two or more of these tools can greatly improve the detection of accessibility violations.
+
+**Color contrast checkers:** When adding new colors, making sure they are accessible is also important. Here are some tools which can help with color contrast testing
+
+- `Accessible web color-contrast checker <https://accessibleweb.com/color-contrast-checker/>`_
+- `Colorsafe generator <https://colorsafe.co/>`_
+
+**Accessibility linters:** If you use VSCode, you may install the `axe accessibility linter <https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter>`_
+which can help you catch and fix potential issues during development.
+
+**Accessibility testing:** Our e2e testing suite which is based on Cypress supports the addition of accessibility tests using `axe-core <https://github.com/dequelabs/axe-core>`_ 
+and `cypress-axe <https://github.com/component-driven/cypress-axe>`_. A custom Cypress command `cy.checkAccessibility`, can also be used directly. 
+This is a great way to prevent accessibility regressions on high impact components.
+
+Tests can be found under the `a11y folder <./src/pybind/mgr/dashboard/frontend/cypress/integration/a11y>`_ in the dashboard. Here is an example:
+
+.. code:: TypeScript
+
+  describe('Navigation accessibility', { retries: 0 }, () => {
+    const shared = new NavigationPageHelper();
+  
+    beforeEach(() => {
+      cy.login();
+      Cypress.Cookies.preserveOnce('token');
+      shared.navigateTo();
+    });
+  
+    it('top-nav should have no accessibility violations', () => {
+      cy.injectAxe();
+      cy.checkAccessibility('.cd-navbar-top');
+    });
+  
+    it('sidebar should have no accessibility violations', () => {
+      cy.injectAxe();
+      cy.checkAccessibility('nav[id=sidebar]');
+    });
+  
+});
+
+**Additional guidelines:** If you're unsure about which UI pattern to follow in order to implement an accessibility fix, `patternfly <https://patternfly.org/v4/>`_ guidelines can be used.
```

Comment:
I'd add a short list (~5) of "things to check" (things that you've done so far in your PRs), and which are quick wins:
* Add ARIA labels and descriptions (link to ARIA labels) to actionable HTML elements,
* Don't forget to tag ARIA labels/descriptions or any user-readable text for translation (`i18-title`, `i18n-aria-label`...).
* Add ARIA roles (link to ARIA roles) to tag HTML elements that behave different from their intended behaviour (`<a>` tags behaving as `<buttons>`) or that provide extended behaviours (roles),
* Avoid poor color contrast choices (foreground-background) when styling a component (link to the afore-mentioned color contrast checker tool).
* ...

Reply:
Thanks @epuertat this is quite a list of easily missed items